src/api/server.rs

```diff
@@ -59,11 +57,9 @@ impl Api for LocalServer {
 
     async fn region(&self, _request: Request<RegionReq>) -> ApiResult<RegionRes> {
         let region_params = self.region_watch.borrow();
-        let region = region_params
-            .as_ref()
-            .map_or_else(|| self.default_region, |params| params.region)
-            .into();
-        Ok(Response::new(RegionRes { region }))
+        Ok(Response::new(RegionRes {
```

Comment:
I see no error path. Why return a `Result`?

Reply:
To meet the tonic  contract